menpo/image/base.py

```diff
@@ -15,7 +15,12 @@
                              transform_about_centre)
 from menpo.visualize.base import ImageViewer, LandmarkableViewable, Viewable
 
-from .interpolation import scipy_interpolation, cython_interpolation
+from .interpolation import scipy_interpolation
+try:
+    from .interpolation import cython_interpolation
+except ImportError:
```

Comment:
IOError?

Reply:
`ImportError` should be fine, typo in the other diff